lib/readymade/controller/serialization.rb

```diff
@@ -21,15 +21,12 @@ def record_response(record, *args)
       end
 
       def collection_response(collection, *args)
-        options = args.extract_options!
-
-        render_json(
-          {
-            (options.delete(:root).presence || :items) => serialize_collection(paginate(collection, options), options),
-            count: collection.count
-          },
-          options[:status] || :ok
-        )
+        args.extract_options!.then do |options|
+          render_json(
+            serialize_collection(paginate(collection, options), options).merge(count: collection.count),
```

Comment:
not sure that `count` is merged in the same place - is the collection_response same as if was?

Reply:
@OrestF
I will double-check that

Reply:
`serialize_collection(paginate(collection, options), options)` will return a hash, so we can safely merge `count`